packages/login/src/components/LoginForm/LoginForm.js

```diff
@@ -12,6 +12,18 @@ import {
 import {Pages} from '../../types/Pages'
 
 export class LoginForm extends Component {
+  constructor(props) {
+    super(props)
+
+    this.passwordRef = React.createRef()
```

Comment:
looks quite complicated :)

couldn't we simply set the `autoFocus` prop on the password field instead of the username field if `props.username` is set?